### PR TITLE
Integration test for fixed-batch-size model deployment

### LINKED ISSUE #365: Write an integration test for batch prediction
* This should use the new batch prediction functionality exposed by the query frontend.

### PR DESCRIPTION
**Pull request by Corey-Zumar**
* Adds the `batch_size` parameter to relevant deployer APIs

* Fixes #365

### CODE REVIEW

File: clipper_admin/clipper_admin/deployers/pyspark.py [hunk 1]
```diff
@@ -79,13 +80,17 @@ def create_endpoint(
         The number of replicas of the model to create. The number of replicas
         for a model can be changed at any time with
         :py:meth:`clipper.ClipperConnection.set_num_replicas`.
+    batch_size : int, optional
+        The user-defined batch size.
```
Comment (dcrankshaw) [inline clipper_admin/clipper_admin/deployers/pyspark.py]: Can you briefly explain what batching this refers to?

**Review by dcrankshaw — CHANGES_REQUESTED** [review]
Just fix the batch size documentation and this is good to go.

File: clipper_admin/clipper_admin/deployers/pyspark.py (hunk 1 above)
Comment (Corey-Zumar) [reply clipper_admin/clipper_admin/deployers/pyspark.py]: Done!

**Review by dcrankshaw — APPROVED** [review]
LGTM

**Pull request merged**